apps/api/src/app/cache/cache.module.ts

```diff
@@ -8,6 +15,16 @@ import { CacheService } from './cache.service';
 @Module({
   imports: [RedisCacheModule],
   controllers: [CacheController],
-  providers: [CacheService, PrismaService]
+  providers: [
+    AlphaVantageService,
+    CacheService,
+    ConfigurationService,
+    DataGatheringService,
```

Comment:
This is annoying. I only need the `DataGatheringService`, but I have to provide all its dependencies as well 🤔 

Any idea @vzickner?

Reply:
Can you not just import another module which exports the services?

Reply:
> Can you not just import another module which exports the services?

#278